modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/store/FilePageStoreManager.java

```diff
@@ -49,12 +52,24 @@ public class FilePageStoreManager implements PageReadWriteManager {
     /** File suffix. */
     public static final String FILE_SUFFIX = ".bin";
 
+    /** Suffix of the temporary file. */
+    public static final String TMP_FILE_SUFFIX = ".tmp";
+
     /** Partition file prefix. */
     public static final String PART_FILE_PREFIX = "part-";
 
-    /** Partition file template. */
+    /** Partition delta file prefix. */
+    public static final String PART_DELTA_FILE_PREFIX = "part-%d-delta-";
```

Comment:
You could write `PART_FILE_PREFIX + ...`, why duplicating these strings?

Reply:
fxi it